flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -254,9 +254,10 @@ class FMCMainDisplay extends BaseAirliners {
         this.updateFuelVars();
         this.updatePerfSpeeds();
 
-        CDUPerformancePage.UpdateThrRedAccFromOrigin(this, true, true);
+        CDUPerformancePage.UpdateThrRedAccFromOrigin(this);
         CDUPerformancePage.UpdateEngOutAccFromOrigin(this);
         SimVar.SetSimVarValue("L:AIRLINER_THR_RED_ALT", "Number", this.thrustReductionAltitude);
+        SimVar.SetSimVarValue("L:A32NX_ENG_OUT_ACC_ALT", "feet", this.engineOutAccelerationAltitude);
```

Comment:
I wondered at these myself. The are repeated in line 512-513. Without knowing if the simVar needs to initialised early I added in the var L:A32NX_ENG_OUT_ACC_ALT  as the two simVars go together.  I would think that line 259 and 260 could be deleted. I have run a build without them, but lost my nerve on submitting the PR with them deleted in case they are required and I could not find where from.

Reply:
Having thought about the inputs. It is possible that 1200/q200 could be entered. If the if block separated the two parts 1200 would be past as valid. When in fact the whole line should be returned as invalid format.

Reply:
I deleted these two lines no adverse effect seen.